testsuite/forge/src/interface/system_metrics.rs

```diff
@@ -86,7 +92,8 @@ fn ensure_metrics_threshold(
     let breach_pct = (breach_count * 100) / metrics.len();
     if breach_pct > threshold.max_breach_pct {
         bail!(
-            "Metrics violated threshold, max_breach_pct {:?}, breach_pct{:?} ",
+            "{:?} metrics violated threshold, max_breach_pct: {:?}, breach_pct: {:?} ",
```

Comment:
let's add the threshold value to the message (and I think metrics should be in singular, metric):
```
            "{:?} metric violated threshold of {}, max_breach_pct: {:?}, breach_pct: {:?} "
            metrics_name, 
            threshold.max
            threshold.max_breach_pct,
            breach_pct,


```